source/code-generation/query-converter/convert-queries.txt

```diff
@@ -0,0 +1,158 @@
+
+      a. Copy your SQL query to your clipboard.
+      #. Under the :guilabel:`SQL Query` header, paste your SQL query
+         into the text box.
+   
+   .. step:: Convert and test syntax
+
+      a. Click the :guilabel:`Convert` button. Wait for your SQL code to 
+         converted. The converted MongoDB syntax code displays 
+         in the :guilabel:`Converted MongoDB Query` text box.
+      #. Click the :icon-fa5:`copy` icon to copy the MongoDB syntax 
+         to your clipboard.
+      #. Test and validate the generated MongoDB syntax.
+
+      .. tip:: 
+
+         .. include:: /includes/fact-query-converter-filter.rst
+
+Examples
+--------
+
+The following table shows examples of SQL queries converted to MongoDB 
+syntax using query converter:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - SQL Syntax
+     - MongoDB Syntax
+
+   * - 
+       .. code-block:: sql
+          :copyable: false
+
+          SELECT * 
+          FROM CUSTOMERS
+     - 
+       .. code-block:: 
+          :copyable: false
+
+          async function query(db) {
+            return await db.collection('CUSTOMERS').find({
+            }).toArray();
+          }
+
+   * -
+      .. code-block:: sql
+         :copyable: false
+
+         SELECT CONTACTNAME, CITY 
+         FROM CUSTOMERS 
+         WHERE CONTACTNAME LIKE '%SMITH%'
+
+     - 
+      .. code-block:: node
+         :copyable: false
+
+         async function query(db) {
+         return await db.collection('CUSTOMERS').find({
+               CONTACTNAME: { $regex: '.*SMITH.*' }
+            }, {
+               projection: { CONTACTNAME: 1, CITY: 1, _id: 0 }
+            }).toArray();
+         }
+
+   * -
+      .. code-block:: sql
+         :copyable: false
+
+         SELECT CUSTOMERID, CITY
+         FROM CUSTOMERS AS C
+         JOIN ORDERS AS O
+            ON C.CUSTOMERID = O.CUSTOMERID
+         WHERE CONTACTNAME IN('ABI','JIM')
+
+     -
+      .. code-block:: node
+         :copyable: false
+
+         const query = async (db) => {
+         return await db.collection('Customers').aggregate([
+            {
+               $lookup: {
+               from: 'Orders',
+               localField: 'CUSTOMERID',
+               foreignField: 'CUSTOMERID',
+               as: 'customer_orders'
+               }
+            },
+            {
+               $match: {
+               CONTACTNAME: { $in: ['ABI', 'JIM'] }
+               }
+            },
+            {
+               $project: {
+               CUSTOMERID: 1,
+               CITY: 1
+               }
+            }
+            ]).toArray();
+         };
+
+Next Steps
+----------
+
+- :ref:`rm-convert-views`
+- :ref:`rm-convert-stored-procedures`
+
+Learn More
+----------
+
+- `SQL Comparison Chart <https://www.mongodb.com/docs/manual/reference/sql-comparison/>`_
+- `SQL to Aggregation Mapping Chart <https://www.mongodb.com/docs/manual/reference/sql-aggregation-comparison/>`_
```

Comment:
[question]

Can we use a ref for these? If no ref is available, we should use the `:manual:` docs role.

Reply:
Good call, updated.